### DISCUSSION TITLE
Hit sound not played properly

### DISCUSSION THREAD

**Discussion by UndefFox** [ppy/osu#24572] [Q&A]
### Type

Game behaviour

### Bug description

Maps that have custom hit sound don't play sound at the right bars. 

### Screenshots or videos

for example, map "Piano Beatmap Set - Canon 4k" on both versions:

https://github.com/ppy/osu/assets/86872388/3cfa3b64-fa51-402b-ab4f-63db8ef59c09

https://github.com/ppy/osu/assets/86872388/cf24d611-a8d6-4c0a-807a-640a62183597

### Version

2023.811.0

### Logs

[runtime.log](https://github.com/ppy/osu/files/12327255/runtime.log)

**Comment by UndefFox** [comment]
This problem only appears in the Windows build. Linux lazer build is working properly.

**Comment by peppy** [comment]
Probably a user offset issue. Doesn't help that there's no link to the beatmap in question. The video looks fine apart from audio playing earlier than video (which is basically impossible, so maybe a recording issue).

**Comment by UndefFox** [comment]
Map in the videos: https://osu.ppy.sh/beatmapsets/332673#mania/1328823

The problem that lazer version playing wrong sounds. On basic version you can clearly hear good melody. Lazer version playing wrong sounds so melody is messed up.

**Reply by peppy** [reply]
So I can't hear any sound in your video, but I've just tested locally and the beatmap sounds fine here. You're going to have to provide more details.

**Comment by UndefFox** [comment]
Update on current status. Bug is still persists on Windows build of Lazer, but with small difference: hit sounds are no longer missalign, but some notes are still miss it. Bug is only present in Windows build, AppImage doesn't have such a problem. Here new videos showing the issue:

Linux (AppImage, all hits can be clearly heard):

[LinuxC.webm](https://github.com/user-attachments/assets/77d53287-c1d6-4642-8ee9-4281058bb457)

Windows (half cords, a.e. 3 or more notes pressed at the same time, don't play all the sounds, lack of bass note is clearly heard from 9 seconds into the level):

[WindowsC.webm](https://github.com/user-attachments/assets/e4fc709a-a5da-4d3e-a2f3-5ee24600c92f)

*Uploading in webm, hope this time sound won't be an issue*
